indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/template/RepositoryDoc.java

```diff
@@ -30,14 +30,12 @@
 import com.google.api.services.cloudsearch.v1.model.RepositoryError;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
-import com.google.common.util.concurrent.AsyncFunction;
-import com.google.common.util.concurrent.Futures;
-import com.google.common.util.concurrent.ListenableFuture;
-import com.google.common.util.concurrent.MoreExecutors;
+import com.google.common.util.concurrent.*;
```

Comment:
Please refraining from using wildcard imports.
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Agreed. I had to fix this in my IDE - it was doing it automatically below a certain threshold.